_layouts/job_post.html

```diff
@@ -3,70 +3,68 @@
+    <p>
         <em>
-          In keeping with our beliefs and goals, no employee or applicant will face
-          discrimination or harassment based on race, color, ancestry, national
-          origin, religion, education, age, gender identity, sexual orientation,
-          marital domestic partner status, familial status, disability status, or
-          veteran status.
+            In keeping with our beliefs and goals, no employee or applicant will face
+            discrimination or harassment based on race, color, ancestry, national
+            origin, religion, education, age, gender identity, sexual orientation,
+            marital domestic partner status, familial status, disability status, or
+            veteran status.
         </em>
-      </p>
-      <p>
+    </p>
+    <p>
         <em>
-          We will consider for employment qualified applicants with arrest and
-          conviction records.
-          <a
+            We will consider for employment qualified applicants with arrest and
+            conviction records.
+            <a
             href="https://bantheboxcampaign.org"
             target="_blank"
             rel="noopener noreferrer"
-          >
+            >
             #banthebox
-          </a>
-        </em>
-      </p>
+        </a>
+    </em>
+</p>
 
-      <div>
-        <a class="monospace lg-link" href="/jobs">Back to Jobs</a>
-      </div>
-    </div>
-  </div>
+<div>
+    <a class="monospace lg-link" href="/jobs">Back to Jobs</a>
+</div>
+</div>
 </div>
 
 <script
-  src="https://cdn.jsdelivr.net/npm/luxon@3.3.0/build/global/luxon.min.js"
-  crossorigin="anonymous"
+src="https://cdn.jsdelivr.net/npm/luxon@3.3.0/build/global/luxon.min.js"
+crossorigin="anonymous"
 ></script>
 <script>
-  // override default Timezone for California
-  luxon.Settings.defaultZone = "America/Los_Angeles";
-  const DateTime = luxon.DateTime;
-  const now = DateTime.now();
+    // override default Timezone for California
+    luxon.Settings.defaultZone = "America/Los_Angeles";
+    const DateTime = luxon.DateTime;
+    const now = DateTime.now();
 
-  const job = {% include job-json.html job=page %};
-  const open_date = DateTime.fromISO(job.open_date).startOf("day");
-  const close_date = DateTime.fromISO(job.close_date).endOf("day");
+    const job = {% include job-json.html job=page %};
+    const open_date = DateTime.fromISO(job.open_date).startOf("day");
+    const close_date = DateTime.fromISO(job.close_date).endOf("day");
 
-  if(open_date > now || close_date < now) {
-    const apply = document.getElementById("apply");
-    if (apply) {
-      apply.classList.add("btn", "disabled");
-      apply.setAttribute("aria-disabled", "true");
+    if(open_date > now || close_date < now) {
+        const apply = document.getElementById("apply");
+        if (apply) {
+            apply.classList.add("btn", "disabled");
+            apply.setAttribute("aria-disabled", "true");
+        }
     }
-  }
 </script>
```

Comment:
Re-indented this page as 4 space.

Reply:
This app doesn't have a `pyproject.toml` file or anything setting the indentation (_I think_).

Reply:
I saw your comment on the clean up issue #123 

I think we want a `.prettierrc` file that specifies the spacing, since Prettier is our default formatter for HTML, JS, CSS -- something like:

```json
{
  "tabWidth": 2,
  "useTabs": false,
  "printWidth": 120
}
```

Reply:
Can do this is a separate PR. The indentation on some of these HTML files are extremely messed up.

Reply:
> Re-indented this page as 4 space.

Hmm I thought we used 2 though?

Reply:
I thought we used 2 too, but NONE of the pages at least..when I look at my VS Code use 2 spaces. Getting this on all pages:
<img width="354" alt="image" src="https://github.com/compilerla/compiler.la/assets/3673236/0e6e52e1-6527-41de-aa8a-49e3e5700f8c">

But that confused me b/c apparently Prettier's default is 2: https://prettier.io/docs/en/options.html#tab-width

So I'm not really sure where the 4 is coming from.